### ISSUE TITLE
about Vulkan

### ISSUE THREAD

**Issue by mingkosan** [substring/os#54]
I have installed PCSX2, but when I attempt to launch Vulkan, it prompts that it is not supported. Could this be a system driver issue? The graphics card is an R9370.

**Comment by substring** [comment]
Do you have Vulkan drivers installed ? What GPU ?

**Comment by mingkosan** [comment]
R9 370,I did not install any drivers. After installing this system, I ran pcsx2.appimage, but encountered an issue where Vulkan could not be used. This was because Batocera allowed the selection of Vulkan, so would I need to install a separate Vulkan driver?

**Comment by substring** [comment]
This is not Batocera. GroovyArcade is way more basic than Batocera on purpose. It's meant to run GroovyMAME in the best conditions. The rest is up to your linux knowledge

**Comment by mingkosan** [comment]
Indeed, this system is highly accessible, allowing me to install additional emulators on it which run seamlessly on CRT. I have also treated it as a Linux system, thereby acquiring substantial knowledge about Linux.

Additionally, there is an RGBuntu system that is quite commendable, albeit somewhat outdated, with its kernel too old to be updated. For new versions to emerge, developers would have to release them. This system, however, allows for the execution of some Linux games and PC games, and one can even configure it to run at 240p.
